README.md

```diff
@@ -2,7 +2,7 @@
 <p style="font-weight:600; font-size:36px">Vega Strike Space Simulation Engine</p>
 </header>
 
-[![Build Status](https://travis-ci.org/vegastrike/Vega-Strike-Engine-Source.svg?branch=master)](https://travis-ci.org/vegastrike/Vega-Strike-Engine-Source)
+<!-- ![CI](https://github.com/vegastrike/Vega-Strike-Engine-Source/workflows/gh-actions-pr/badge.svg) -->
```

Comment:
this will probably become a list of all the gh-actions we do that we deem relevant; but this is a good start.